api/room/schema.py

```diff
@@ -79,8 +81,23 @@ def mutate(self, info, office_id, **kwargs):
         # remove block ID from kwargs, can't be saved in rooms model
         if kwargs.get('block_id'):
             kwargs.pop('block_id')
-        room = RoomModel(**kwargs)
+        room = RoomModel(
```

Comment:
Nice work @bevkololi,

Kindly share the reason for explicitly stating the kwargs.

Reply:
This was because when of the relationship table (room_tags). It is treated as a lists when one wants to create the tag, that is they are appended instead. When I try creating it as follows: `room = RoomModel(**kwargs)`, I get the error `int' object has no attribute 'sa_instance_state`. Therefore the solution is to explicitly state all the kwargs to avoid this error. Here is a resource I used: https://github.com/miguelgrinberg/flasky/issues/162